microsite/sidebars.json

```diff
@@ -190,7 +188,6 @@
       "architecture-decisions/adrs-adr010",
       "architecture-decisions/adrs-adr011"
     ],
-    "Contribute": ["../CONTRIBUTING"],
```

Comment:
I think we want this one to work tbh, any idea why it isn't?

Reply:
Not sure, will investigate.